scripts/alert_emails/utils.ts

```diff
@@ -0,0 +1,85 @@
+import path from 'path';
+import fs from 'fs-extra';
+import * as Handlebars from 'handlebars';
+import { Alert } from './interfaces';
+import { Level } from '../../src/common/level';
+
+const thermometerBaseURL =
+  'https://data.covidactnow.org/thermometer_screenshot';
+const unsubscribeURL = 'https://covidactnow.org/alert_unsubscribe';
+
+const alertTemplate = Handlebars.compile(
+  fs.readFileSync(path.join(__dirname, 'template.html'), 'utf8'),
+);
+
+interface AlertTemplateData {
+  change: string;
+  location_name: string;
+  img_alt: string;
+  img_url: string;
+  last_updated: string;
+  location_url: string;
+  unsubscribe_link: string;
+  feedback_subject_line: string;
+}
+
+function generateAlertEmailContent(
+  emailAddress: string,
+  locationAlert: Alert,
+): string {
+  const {
+    newLevel,
+    oldLevel,
+    locationName,
+    lastUpdated,
+    locationURL,
+  } = locationAlert;
+
+  const data: AlertTemplateData = {
+    change: 'new risk score',
```

Comment:
This is temporary for this one send, right?  Can you add a TODO or else just make sure to remember to put this back?